Panic in github.com/prysmaticlabs/prysm/beacon-chain/blockchain/forkchoice.(*Store).updateAttVotes

```
{"error":"could not verify indexed attestation: attestation aggregation signature did not verify","message":"Removing attestation from queue.","prefix":"forkchoice","severity":"WARNING"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xcb9f17]

goroutine 212 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/blockchain/forkchoice.(*Store).updateAttVotes(0xc0003dc900, 0x188b320, 0xc007073650, 0x0, 0xc0049555c0, 0x20, 0x20, 0x1936, 0xc007073650, 0xc0043b1b00)
        beacon-chain/blockchain/forkchoice/process_attestation.go:182 +0x37
github.com/prysmaticlabs/prysm/beacon-chain/blockchain/forkchoice.(*Store).OnAttestation(0xc0003dc900, 0x188b320, 0xc007073650, 0xc000d1a2a0, 0x0, 0x0)
        beacon-chain/blockchain/forkchoice/process_attestation.go:96 +0x603
github.com/prysmaticlabs/prysm/beacon-chain/blockchain.(*Service).ReceiveAttestationNoPubsub(0xc000142380, 0x188b320, 0xc007073620, 0xc000d1a2a0, 0x0, 0x0)
        beacon-chain/blockchain/receive_attestation.go:30 +0x119
github.com/prysmaticlabs/prysm/beacon-chain/blockchain.(*Service).processAttestation(0xc000142380)
        beacon-chain/blockchain/receive_attestation.go:68 +0x246
created by github.com/prysmaticlabs/prysm/beacon-chain/blockchain.(*Service).Start
        beacon-chain/blockchain/service.go:141 +0x4aa
```

Experienced this a little while ago in canary running image tag `HEAD-3b18ae` @ commit 3b18aee18171c290991f0547000d29194a6d579f

--- a/beacon-chain/blockchain/forkchoice/process_attestation.go
+++ b/beacon-chain/blockchain/forkchoice/process_attestation.go
@@ -89,7 +89,7 @@ func (s *Store) OnAttestation(ctx context.Context, a *ethpb.Attestation) error {
 	// Use the target state to to validate attestation and calculate the committees.
 	indexedAtt, err := s.verifyAttestation(ctx, baseState, a)
 	if err != nil {
-		log.WithError(err).Warn("Removing attestation from queue.")
+		return err
 	}
 
 	// Update every validator's latest vote.

